web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerWmsGetCapabilities.java

```diff
@@ -0,0 +1,52 @@
+package org.fao.geonet.monitor.link;
+
+import org.apache.log4j.Logger;
+import org.jdom.Element;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class LinkCheckerWmsGetCapabilities extends LinkCheckerDefault {
+
+    private static Logger logger = Logger.getLogger(LinkCheckerWmsGetCapabilities.class);
+
+    @Override
+    public void setOnlineResource(String uuid, final Element onlineResource) {
+        super.setOnlineResource(uuid, onlineResource);
+        url += "?service=wms&version=1.1.1&request=GetCapabilities";
+    }
+
+    @Override
+    public boolean check() {
+
+        try {
+            HttpURLConnection connection;
+            connection = (HttpURLConnection) (new URL(url)).openConnection();
+            connection.setConnectTimeout(LinkMonitorService.timeout * 1000);
+            connection.setReadTimeout(LinkMonitorService.timeout * 1000);
+            connection.setRequestMethod("GET");
+            connection.connect();
+
+            if(connection.getResponseCode() != 200) {
+                logger.info(String.format("link broken uuid='%s', url='%s', error='bad response code %d'",
+                        this.uuid, this.url, connection.getResponseCode()));
+                return false;
+            }
+
+            if (connection.getContentType().equals("application/vnd.ogc.wms_xml")) {
+                LinkCheckerUtils.parseXML(connection.getInputStream());
+                return true;
+            } else {
+                logger.info(String.format("link broken uuid='%s', url='%s', error='unexpected content-type'", this.uuid, this.url));
+                return false;
+            }
+
+        } catch (Exception e) {
+
+            logger.info(String.format("link broken uuid='%s', url='%s', error='%s' stack='%s'",
+                    uuid, url, e.getMessage(), LinkCheckerUtils.exceptionToString(e)));
```

Comment:
I can see this stuff changing when the sumo side work is done. I've left the = and quoted the exception to hopefully make it easier to parse. I also quite like having the entire logged output handled in the single logger method rather than split over two lines, which could create race conditions with other logging actions. It would be a minor change to make after the sumo scripting side has been confirmed to work.